tdms/tests/system/utils.py

```diff
@@ -38,7 +29,7 @@ class HDF5File(dict):
 
     COMPLEX_TYPE_STRING = r"[('real', '<f8'), ('imag', '<f8')]"
 
-    def __init__(self, filepath: Path):
+    def __init__(self, filepath: Union[str, Path]):
```

Comment:
Out of curiosity, are these type annotations actually checked at any point?

Reply:
Not checked, just highlighted by PyCharm